core/common/src/main/java/alluxio/grpc/GrpcServerBuilder.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0 (the
+ * "License"). You may not use this work except in compliance with the License, which is available
+ * at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.grpc;
+
+import alluxio.security.authentication.AuthenticationServer;
+import alluxio.security.authentication.DefaultAuthenticationServer;
+import alluxio.util.SecurityUtils;
+import io.grpc.BindableService;
+import io.grpc.ServerInterceptor;
+import io.grpc.ServerInterceptors;
+import io.grpc.ServerServiceDefinition;
+import io.grpc.netty.NettyServerBuilder;
+
+import java.net.InetSocketAddress;
+import java.util.concurrent.Executor;
+
+import javax.annotation.Nullable;
+
+/**
+ * Provides authenticated gRPC server creation.
+ */
+public class GrpcServerBuilder {
+
+  NettyServerBuilder mNettyServerBuilder;
```

Comment:
can these be `private` and `final`?

Reply:
can be private, but can't be final due to beautified setters.